constraints.txt

```diff
@@ -1,3 +1,2 @@
 rpc-qtest-swagger-client<10.0.0
-pytest-rpc<1.0.0
```

Comment:
Should there be no pin whatsoever?

Reply:
Yeah It should probably keep the less than 1.0 and add a lower boundary.